tutorlimesurvey/__about__.py

```diff
@@ -1 +1 @@
-__version__ = "15.0.0"
+__version__ = "15.1.0"
```

Comment:
why minor?

Reply:
The XBlock was added as extra pip requirement. Or change it to patch?

Reply:
That's fien